.github/workflows/cd.yml

```diff
@@ -0,0 +1,17 @@
+name: cd
+on:
+  workflow_run:
+    workflows: ["ci"]
+    types:
+      - completed
+  push:
+    tags:
+      - "v[0-9]+.[0-9]+.[0-9]+"
```

Comment:
What about RC releases like https://github.com/mattermost/mattermost-plugin-github/releases/tag/v2.1.3-rc2? Shouldn't CI/CD also trigger for them?

Reply:
This is a great point we can definitely evolve this.

Reply:
Is there a ticket for fix this issue?